.evergreen/config.yml

```diff
@@ -827,20 +827,18 @@ functions:
 
   start-kms-mock-server:
     - command: shell.exec
-      type: test
```

Comment:
Removing `type: test` seems right here. The default `command_type` on L13 is `setup`. If this task fails it will indicate a setup failure, rather than a test failure (https://github.com/evergreen-ci/evergreen/wiki/Project-Configuration-Files#command-failure-colors)

Reply:
Yeah `setup` definitely seems like the right type; not sure why I had `test` before.